prov/rxm/src/rxm.h

```diff
@@ -72,13 +72,16 @@
 
 #define RXM_IOV_LIMIT 4
 
-#define RXM_MR_MODES	(OFI_MR_BASIC_MAP | FI_MR_LOCAL)
+#define RXM_MR_MODES	(OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_RAW)
 #define RXM_MR_VIRT_ADDR(info) ((info->domain_attr->mr_mode == FI_MR_BASIC) ||\
 				info->domain_attr->mr_mode & FI_MR_VIRT_ADDR)
 
 #define RXM_MR_PROV_KEY(info) ((info->domain_attr->mr_mode == FI_MR_BASIC) ||\
 			       info->domain_attr->mr_mode & FI_MR_PROV_KEY)
 
+#define RXM_MR_RAW(info) ((info->domain_attr->mr_mode & FI_MR_RAW) ||\
+			   info->domain_attr->mr_key_size > sizeof(uint64_t))
```

Comment:
A key size larger than 64 bits requires FI_MR_RAW.  mr_mode & MR_RAW should be sufficient.